gcc/common/config/riscv/riscv-common.cc

```diff
@@ -1230,6 +1232,7 @@ static const riscv_ext_flag_table_t riscv_ext_flag_table[] =
 
   {"xcorev",    &gcc_options::x_riscv_xcorev_flags, MASK_XCOREV},
   {"xcorevelw", &gcc_options::x_riscv_xcorev_flags, MASK_XCOREVELW},
+  {"xcorevmac", &gcc_options::x_riscv_xcorev_flags, MASK_XCOREVMAC},
```

Comment:
This should be `xcvmac`.    See the  [agreed specification](https://github.com/openhwgroup/core-v-sw/blob/master/specifications/corev-isa-extension-naming.md).

Reply:
See comments above about making this change later.